docs/sites.yml

```diff
@@ -5589,3 +5589,52 @@
     - Marketing
     - Blog
   featured: false
+- title: NH Hydraulikzylinder
+  main_url: https://nh-hydraulikzylinder.com
+  url: https://nh-hydraulikzylinder.com
+  description: >
+    High quality & high performance hydraulic cylinders manufactured in Austria based on the clients requirements
+  categories:
+    - Business
+    - Responsive Design
+  built_by: MangoART
+  built_by_url: https://www.mangoart.at
+  featured: false
+
+- title: Frauennetzwerk Linz-Land
+  main_url: https://frauennetzwerk-linzland.net
+  url: https://frauennetzwerk-linzland.net
+  description: >
+    Homepage for the local women's association providing support to people in need offline and online (Livechat integration)
+  categories:
+    - Nonprofit
+    - Accessibility
+    - Responsive Design
+  built_by: MangoART
+  built_by_url: https://www.mangoart.at
+  featured: false
+
+- title: Mein Traktor
+  main_url: http://www.mein-traktor.at/
+  url: http://www.mein-traktor.at/
+  description: >
+    Homepage of a the main importer of SAME and Lamborghini Tractors in Austria with customer support area
+  categories:
+    - Agriculture
+    - Business
+    - App
+  built_by: MangoART
+  built_by_url: https://www.mangoart.at
+  featured: false
+
+- title: Lamborghini Traktoren
+  main_url: https://lamborghini-traktor.at
```

Comment:
`https://lamborghini-traktor.at` gives me a security issue. Could you fix your certificate?